### ISSUE TITLE
Metadata daemon user not accounted for in split-user mode

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1005]
Originally reported of Google Code with ID 949.

```
<b>What steps will reproduce the problem?</b>
1. Run Ganeti 2.12 in split-user mode with the metadata daemon enabled.
2. Ganeti complains about missing user; provided tools do not create user.

The Ganeti tooling needs to create all the right users / acknowledge the metadata daemon's existence in split-user mode.
```
Originally added on 2014-09-17 17:10:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Apart from the user creation script, it is also not accounted for in daemon-util.
```
Originally added on 2014-09-23 11:26:26 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
In 2.16, the metadata daemon can be started using a separate user.

commit 46f6fb34285f51b151992ddaf3aae5fdc36252c6
Author: Hrvoje Ribicic <riba@google.com>
Date:   Thu May 28 22:16:44 2015 +0000

    Introduce separate user for metad with port capabilites
    
    The metadata daemon was previously running as root due to its need to
    open port 80 to provide information to instances. To allow the daemon
    to run in a more secure way, this patch adds a separate metadata user,
    and grants the metad executable the CAP_NET_BIND_SERVICE capability.
    As a result, the metadata daemon can use the port 80 without having to
    acquire the full set of root capabilities and drop it later.
    
    Signed-off-by: Hrvoje Ribicic <riba@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>


```
Originally added on 2015-05-29 12:09:22 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:38:33 +0000 UTC.

Changed State: Released
